spec/rails_helper.rb

```diff
@@ -16,6 +16,7 @@
 
 RSpec.configure do |config|
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include SignInHelper, type: :controller
```

Comment:
Not needed

Reply:
I tried to remove it, but then the sign_in_helper.rb file wasn't recognized by the controllers.
![image](https://github.com/mla-wtag/ml_Journey/assets/141820916/403af991-9383-40d8-af90-1b70544dc440)

Reply:
Can you try just keeping the method and removing the module?
https://github.com/mla-wtag/ml_Journey/pull/6#discussion_r1350111463 
Let me know if the issue persists.